src/data/eventFiltering.js

```diff
@@ -35,6 +35,5 @@ export function populateEventFilterFrom(queryStringData) {
 }
 
 export function isParticipant(event) {
```

Comment:
Does this function need to be exported?

Reply:
Yes, it's also used in pages/events/index.svelte.

Reply:
Oh, ok, nevermind.